src/main/java/org/minions/devfund/richard/battleship/Ocean.java

```diff
@@ -0,0 +1,99 @@
+package org.minions.devfund.richard.battleship;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Class.
+ */
+public class Ocean {
+
+    private static final int OCEAN_LENGTH = 20;
+    private static final int SHIPS_QUANTITY = 13;
+
+    private Ship[][] ships;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+
+    /**
+     * Ocean.
+     */
+    Ocean() {
+        ships = new Ship[OCEAN_LENGTH][OCEAN_LENGTH];
+        Arrays.stream(ships).forEach(row -> Arrays.fill(row, new EmptySea()));
+    }
+
+    /**
+     * place all Ships randomly.
+     */
+    void placeAllShipsRandomly() {
+        List<Ship> shipsList = Arrays.asList(new BattleShip(), new BattleCruiser(), new Cruiser(), new Cruiser(),
+                new LightCruiser(), new LightCruiser(), new Destroyer(), new Destroyer(), new Destroyer(),
+                new Submarine(), new Submarine(), new Submarine(), new Submarine());
```

Comment:
This should be defined as static.
```java
    private static final List<Ship> SHIP_LIST = Arrays.asList(
            new BattleShip(), 
            new BattleCruiser(), 
            new Cruiser(), 
            new Cruiser(),
            new LightCruiser(), 
            new LightCruiser(), 
            new Destroyer(), 
            new Destroyer(), 
            new Destroyer(),
            new Submarine(), 
            new Submarine(), 
            new Submarine(), 
            new Submarine()
    );
```

However this is not a good solution
I recommend you to use a Map